backend/server/src/server.rs

```diff
@@ -142,37 +141,37 @@ impl Service for AdServer {
                 let restful = RegexSet::new(&[
                     // rudimentary routing. ORDER MATTERS. And we're using the index of these as
                     // the key for match below.
+                    r"^/facebook-ads/?$",
+                    r"^/facebook-ads/admin/?$",
```

Comment:
Hey @thejefflarson What's the impetus for this change? I had been routing everything under `/facebook-ads//admin/` to `admin.html` so that React could do the routing (e.g. to the Grouped Attributes pages).

Reply:
Ugh. I'm so sorry, I misunderstood what was going on there, because we weren't matching on &0, and I wasn't thinking it all the way through. Just so I understand what we're doing we need these routes right?

ads
ads/(\d+)
admin
admin/ads/(\d+)
admin/targets
admin/segments

Is that right? I can make this change bACK but do you want to make it?

Reply:
I got the change. One sec.